frontend/public/components/notification-drawer.tsx

```diff
@@ -229,7 +241,10 @@ export const ConnectedNotificationDrawer_: React.FC<ConnectedNotificationDrawerP
       poll(`${prometheusBaseURL}/${PrometheusEndpoint.RULES}`, 'notificationAlerts', getAlerts);
     } else {
       store.dispatch(
-        UIActions.monitoringErrored('notificationAlerts', new Error('prometheusBaseURL not set')),
+        UIActions.monitoringErrored(
+          'notificationAlerts',
+          new Error(t('notification-drawer~prometheusBaseURL not set')),
```

Comment:
Is this message actually shown in the UI? If not, we don't need to localize.

Reply:
@spadgett this is returned as loadError from the UI store which is displayed in the component AlertErrorState when an error occurs and no alerts are loaded.